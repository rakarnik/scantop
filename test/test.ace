Motif 1
AAAAACGTAAA	0	0	1
ACCCACGTCCC	0	0	1
AGGGACGTGGG	0	0	1
ATTTACGTTTT	0	0	1
*   ****
