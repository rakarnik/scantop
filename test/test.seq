>Seq0
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>Seq1
NNNNNNNNNNNNNNNNNNNNNNNNNNNACTGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>Seq2
AAAAAAAAAAAAAAAAAAAAAAAAAAAACTGAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
>Seq3
TTTTTTTTTTTTTTTTTTTTTTTTTTTACTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT
