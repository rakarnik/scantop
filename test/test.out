Seq2	23	30	1	AAAAACTG	5.38235
Seq3	27	34	0	ACTGTTTT	5.38235
Seq3	28	35	0	CTGTTTTT	5.38235
Seq1	27	34	0	ACTGNNNN	5.13431
Seq1	23	30	1	NNNNACTG	5.13431
Seq1	24	31	1	NNNACTGN	4.88627
Seq1	28	35	0	CTGNNNNN	4.88627
Seq2	27	34	0	ACTGAAAA	3.77291
Seq2	24	31	1	AAAACTGA	3.77291
Seq3	23	30	1	TTTTACTG	3.77291
Seq3	24	31	1	TTTACTGT	3.77291
Seq1	30	37	0	GNNNNNNN	2.31911
Seq1	21	28	0	NNNNNNAC	2.31911
Seq1	23	30	0	NNNNACTG	2.31911
Seq1	30	37	1	GNNNNNNN	2.31911
Seq1	28	35	1	CTGNNNNN	2.31911
Seq1	26	33	1	NACTGNNN	2.31911
Seq2	28	35	0	CTGAAAAA	2.16348
Seq2	22	29	1	AAAAAACT	1.70184
Seq1	22	29	1	NNNNNACT	1.20576
Seq1	26	33	0	NACTGNNN	1.20576
Seq1	21	28	1	NNNNNNAC	0.957717
Seq1	25	32	1	NNACTGNN	0.957717
Seq1	25	32	0	NNACTGNN	0.957717
Seq1	29	36	0	TGNNNNNN	0.957717
Seq1	22	29	0	NNNNNACT	0.248041
Seq1	27	34	1	ACTGNNNN	0.248041
Seq3	26	33	0	TACTGTTT	0.0924027
Seq3	29	36	0	TGTTTTTT	0.0924027
Seq2	21	28	1	AAAAAAAC	0.0924027
Seq3	30	37	0	GTTTTTTT	0.0924027
Seq2	26	33	0	AACTGAAA	0.0924027
Seq1	16	23	0	NNNNNNNN	1.90465e-08
Seq1	17	24	1	NNNNNNNN	1.90465e-08
Seq1	17	24	0	NNNNNNNN	1.90465e-08
Seq1	18	25	1	NNNNNNNN	1.90465e-08
Seq1	18	25	0	NNNNNNNN	1.90465e-08
Seq1	19	26	1	NNNNNNNN	1.90465e-08
Seq1	19	26	0	NNNNNNNN	1.90465e-08
Seq1	3	10	0	NNNNNNNN	1.90465e-08
Seq1	3	10	1	NNNNNNNN	1.90465e-08
Seq1	2	9	0	NNNNNNNN	1.90465e-08
Seq1	4	11	1	NNNNNNNN	1.90465e-08
Seq1	4	11	0	NNNNNNNN	1.90465e-08
Seq1	5	12	1	NNNNNNNN	1.90465e-08
Seq1	5	12	0	NNNNNNNN	1.90465e-08
Seq1	6	13	1	NNNNNNNN	1.90465e-08
Seq1	6	13	0	NNNNNNNN	1.90465e-08
Seq1	7	14	1	NNNNNNNN	1.90465e-08
Seq1	7	14	0	NNNNNNNN	1.90465e-08
Seq1	8	15	1	NNNNNNNN	1.90465e-08
Seq1	8	15	0	NNNNNNNN	1.90465e-08
Seq1	9	16	1	NNNNNNNN	1.90465e-08
Seq1	9	16	0	NNNNNNNN	1.90465e-08
Seq1	10	17	1	NNNNNNNN	1.90465e-08
Seq1	10	17	0	NNNNNNNN	1.90465e-08
Seq1	11	18	1	NNNNNNNN	1.90465e-08
Seq1	11	18	0	NNNNNNNN	1.90465e-08
Seq1	12	19	1	NNNNNNNN	1.90465e-08
Seq1	12	19	0	NNNNNNNN	1.90465e-08
Seq1	13	20	1	NNNNNNNN	1.90465e-08
Seq1	13	20	0	NNNNNNNN	1.90465e-08
Seq1	14	21	1	NNNNNNNN	1.90465e-08
Seq1	14	21	0	NNNNNNNN	1.90465e-08
Seq1	15	22	1	NNNNNNNN	1.90465e-08
Seq1	15	22	0	NNNNNNNN	1.90465e-08
Seq1	16	23	1	NNNNNNNN	1.90465e-08
Seq1	42	49	1	NNNNNNNN	1.90465e-08
Seq1	42	49	0	NNNNNNNN	1.90465e-08
Seq1	43	50	1	NNNNNNNN	1.90465e-08
Seq1	43	50	0	NNNNNNNN	1.90465e-08
Seq1	44	51	1	NNNNNNNN	1.90465e-08
Seq1	44	51	0	NNNNNNNN	1.90465e-08
Seq1	45	52	1	NNNNNNNN	1.90465e-08
Seq1	45	52	0	NNNNNNNN	1.90465e-08
Seq1	46	53	1	NNNNNNNN	1.90465e-08
Seq1	46	53	0	NNNNNNNN	1.90465e-08
Seq1	47	54	1	NNNNNNNN	1.90465e-08
Seq1	47	54	0	NNNNNNNN	1.90465e-08
Seq1	48	55	1	NNNNNNNN	1.90465e-08
Seq1	48	55	0	NNNNNNNN	1.90465e-08
Seq1	49	56	1	NNNNNNNN	1.90465e-08
Seq1	49	56	0	NNNNNNNN	1.90465e-08
Seq1	50	57	1	NNNNNNNN	1.90465e-08
Seq1	50	57	0	NNNNNNNN	1.90465e-08
Seq1	51	58	1	NNNNNNNN	1.90465e-08
Seq1	51	58	0	NNNNNNNN	1.90465e-08
Seq1	52	59	1	NNNNNNNN	1.90465e-08
Seq1	52	59	0	NNNNNNNN	1.90465e-08
Seq1	31	38	1	NNNNNNNN	1.90465e-08
Seq1	31	38	0	NNNNNNNN	1.90465e-08
Seq1	32	39	1	NNNNNNNN	1.90465e-08
Seq1	32	39	0	NNNNNNNN	1.90465e-08
Seq1	33	40	1	NNNNNNNN	1.90465e-08
Seq1	33	40	0	NNNNNNNN	1.90465e-08
Seq1	34	41	1	NNNNNNNN	1.90465e-08
Seq1	34	41	0	NNNNNNNN	1.90465e-08
Seq1	35	42	1	NNNNNNNN	1.90465e-08
Seq1	35	42	0	NNNNNNNN	1.90465e-08
Seq1	36	43	1	NNNNNNNN	1.90465e-08
Seq1	36	43	0	NNNNNNNN	1.90465e-08
Seq1	37	44	1	NNNNNNNN	1.90465e-08
Seq1	37	44	0	NNNNNNNN	1.90465e-08
Seq1	38	45	1	NNNNNNNN	1.90465e-08
Seq1	38	45	0	NNNNNNNN	1.90465e-08
Seq1	39	46	1	NNNNNNNN	1.90465e-08
Seq1	39	46	0	NNNNNNNN	1.90465e-08
Seq1	40	47	1	NNNNNNNN	1.90465e-08
Seq1	40	47	0	NNNNNNNN	1.90465e-08
Seq1	41	48	1	NNNNNNNN	1.90465e-08
Seq1	41	48	0	NNNNNNNN	1.90465e-08
Seq0	14	21	0	NNNNNNNN	1.90465e-08
Seq0	15	22	1	NNNNNNNN	1.90465e-08
Seq0	15	22	0	NNNNNNNN	1.90465e-08
Seq0	16	23	1	NNNNNNNN	1.90465e-08
Seq0	16	23	0	NNNNNNNN	1.90465e-08
Seq0	17	24	1	NNNNNNNN	1.90465e-08
Seq0	17	24	0	NNNNNNNN	1.90465e-08
Seq0	18	25	1	NNNNNNNN	1.90465e-08
Seq0	18	25	0	NNNNNNNN	1.90465e-08
Seq0	19	26	1	NNNNNNNN	1.90465e-08
Seq0	19	26	0	NNNNNNNN	1.90465e-08
Seq0	20	27	1	NNNNNNNN	1.90465e-08
Seq0	20	27	0	NNNNNNNN	1.90465e-08
Seq0	21	28	1	NNNNNNNN	1.90465e-08
Seq0	21	28	0	NNNNNNNN	1.90465e-08
Seq0	22	29	1	NNNNNNNN	1.90465e-08
Seq0	22	29	0	NNNNNNNN	1.90465e-08
Seq0	23	30	1	NNNNNNNN	1.90465e-08
Seq0	23	30	0	NNNNNNNN	1.90465e-08
Seq0	24	31	1	NNNNNNNN	1.90465e-08
Seq0	24	31	0	NNNNNNNN	1.90465e-08
Seq0	25	32	1	NNNNNNNN	1.90465e-08
Seq0	25	32	0	NNNNNNNN	1.90465e-08
Seq0	26	33	1	NNNNNNNN	1.90465e-08
Seq0	26	33	0	NNNNNNNN	1.90465e-08
Seq0	27	34	1	NNNNNNNN	1.90465e-08
Seq0	27	34	0	NNNNNNNN	1.90465e-08
Seq0	28	35	1	NNNNNNNN	1.90465e-08
Seq0	0	7	1	NNNNNNNN	1.90465e-08
Seq0	0	7	0	NNNNNNNN	1.90465e-08
Seq0	1	8	1	NNNNNNNN	1.90465e-08
Seq0	1	8	0	NNNNNNNN	1.90465e-08
Seq0	2	9	1	NNNNNNNN	1.90465e-08
Seq0	2	9	0	NNNNNNNN	1.90465e-08
Seq0	3	10	1	NNNNNNNN	1.90465e-08
Seq0	3	10	0	NNNNNNNN	1.90465e-08
Seq0	4	11	1	NNNNNNNN	1.90465e-08
Seq0	4	11	0	NNNNNNNN	1.90465e-08
Seq0	5	12	1	NNNNNNNN	1.90465e-08
Seq0	5	12	0	NNNNNNNN	1.90465e-08
Seq0	6	13	1	NNNNNNNN	1.90465e-08
Seq0	6	13	0	NNNNNNNN	1.90465e-08
Seq0	7	14	1	NNNNNNNN	1.90465e-08
Seq0	7	14	0	NNNNNNNN	1.90465e-08
Seq0	8	15	1	NNNNNNNN	1.90465e-08
Seq0	8	15	0	NNNNNNNN	1.90465e-08
Seq0	9	16	1	NNNNNNNN	1.90465e-08
Seq0	9	16	0	NNNNNNNN	1.90465e-08
Seq0	10	17	1	NNNNNNNN	1.90465e-08
Seq0	10	17	0	NNNNNNNN	1.90465e-08
Seq0	11	18	1	NNNNNNNN	1.90465e-08
Seq0	11	18	0	NNNNNNNN	1.90465e-08
Seq0	12	19	1	NNNNNNNN	1.90465e-08
Seq0	12	19	0	NNNNNNNN	1.90465e-08
Seq0	13	20	1	NNNNNNNN	1.90465e-08
Seq0	13	20	0	NNNNNNNN	1.90465e-08
Seq0	14	21	1	NNNNNNNN	1.90465e-08
Seq0	41	48	1	NNNNNNNN	1.90465e-08
Seq0	41	48	0	NNNNNNNN	1.90465e-08
Seq0	42	49	1	NNNNNNNN	1.90465e-08
Seq0	42	49	0	NNNNNNNN	1.90465e-08
Seq0	43	50	1	NNNNNNNN	1.90465e-08
Seq0	43	50	0	NNNNNNNN	1.90465e-08
Seq0	44	51	1	NNNNNNNN	1.90465e-08
Seq0	44	51	0	NNNNNNNN	1.90465e-08
Seq0	45	52	1	NNNNNNNN	1.90465e-08
Seq0	45	52	0	NNNNNNNN	1.90465e-08
Seq0	46	53	1	NNNNNNNN	1.90465e-08
Seq0	46	53	0	NNNNNNNN	1.90465e-08
Seq0	47	54	1	NNNNNNNN	1.90465e-08
Seq0	47	54	0	NNNNNNNN	1.90465e-08
Seq0	48	55	1	NNNNNNNN	1.90465e-08
Seq0	48	55	0	NNNNNNNN	1.90465e-08
Seq0	49	56	1	NNNNNNNN	1.90465e-08
Seq0	49	56	0	NNNNNNNN	1.90465e-08
Seq0	50	57	1	NNNNNNNN	1.90465e-08
Seq0	50	57	0	NNNNNNNN	1.90465e-08
Seq0	51	58	1	NNNNNNNN	1.90465e-08
Seq0	51	58	0	NNNNNNNN	1.90465e-08
Seq0	52	59	1	NNNNNNNN	1.90465e-08
Seq0	52	59	0	NNNNNNNN	1.90465e-08
Seq1	0	7	1	NNNNNNNN	1.90465e-08
Seq1	0	7	0	NNNNNNNN	1.90465e-08
Seq1	2	9	1	NNNNNNNN	1.90465e-08
Seq1	1	8	1	NNNNNNNN	1.90465e-08
Seq1	1	8	0	NNNNNNNN	1.90465e-08
Seq0	28	35	0	NNNNNNNN	1.90465e-08
Seq0	29	36	1	NNNNNNNN	1.90465e-08
Seq0	29	36	0	NNNNNNNN	1.90465e-08
Seq0	30	37	1	NNNNNNNN	1.90465e-08
Seq0	30	37	0	NNNNNNNN	1.90465e-08
Seq0	31	38	1	NNNNNNNN	1.90465e-08
Seq0	31	38	0	NNNNNNNN	1.90465e-08
Seq0	32	39	1	NNNNNNNN	1.90465e-08
Seq0	32	39	0	NNNNNNNN	1.90465e-08
Seq0	33	40	1	NNNNNNNN	1.90465e-08
Seq0	33	40	0	NNNNNNNN	1.90465e-08
Seq0	34	41	1	NNNNNNNN	1.90465e-08
Seq0	34	41	0	NNNNNNNN	1.90465e-08
Seq0	40	47	0	NNNNNNNN	1.90465e-08
Seq0	40	47	1	NNNNNNNN	1.90465e-08
Seq0	39	46	0	NNNNNNNN	1.90465e-08
Seq0	39	46	1	NNNNNNNN	1.90465e-08
Seq0	38	45	0	NNNNNNNN	1.90465e-08
Seq0	38	45	1	NNNNNNNN	1.90465e-08
Seq0	37	44	0	NNNNNNNN	1.90465e-08
Seq0	37	44	1	NNNNNNNN	1.90465e-08
Seq0	36	43	0	NNNNNNNN	1.90465e-08
Seq0	36	43	1	NNNNNNNN	1.90465e-08
Seq0	35	42	1	NNNNNNNN	1.90465e-08
Seq0	35	42	0	NNNNNNNN	1.90465e-08
Seq1	20	27	1	NNNNNNNA	-1.3614
Seq1	29	36	1	TGNNNNNN	-1.3614
Seq1	20	27	0	NNNNNNNA	-1.3614
Seq1	24	31	0	NNNACTGN	-1.3614
Seq3	25	32	1	TTACTGTT	-1.51704
Seq3	26	33	1	TACTGTTT	-1.51704
Seq2	21	28	0	AAAAAAAC	-1.51704
Seq3	28	35	1	CTGTTTTT	-1.51704
Seq3	30	37	1	GTTTTTTT	-1.51704
Seq2	23	30	0	AAAAACTG	-1.51704
Seq3	22	29	1	TTTTTACT	-1.51704
Seq2	25	32	0	AAACTGAA	-1.51704
Seq3	21	28	0	TTTTTTAC	-1.51704
Seq2	25	32	1	AAACTGAA	-1.51704
Seq2	30	37	1	GAAAAAAA	-1.51704
Seq2	26	33	1	AACTGAAA	-1.51704
Seq3	23	30	0	TTTTACTG	-1.51704
Seq2	28	35	1	CTGAAAAA	-1.51704
Seq3	25	32	0	TTACTGTT	-1.51704
Seq2	29	36	0	TGAAAAAA	-3.12647
Seq3	21	28	1	TTTTTTAC	-3.12647
Seq2	30	37	0	GAAAAAAA	-3.12647
Seq2	46	53	1	AAAAAAAA	-3.58811
Seq2	45	52	1	AAAAAAAA	-3.58811
Seq3	13	20	0	TTTTTTTT	-3.58811
Seq3	0	7	0	TTTTTTTT	-3.58811
Seq2	44	51	1	AAAAAAAA	-3.58811
Seq2	43	50	1	AAAAAAAA	-3.58811
Seq2	52	59	1	AAAAAAAA	-3.58811
Seq2	51	58	1	AAAAAAAA	-3.58811
Seq3	12	19	0	TTTTTTTT	-3.58811
Seq3	1	8	0	TTTTTTTT	-3.58811
Seq3	2	9	0	TTTTTTTT	-3.58811
Seq2	47	54	1	AAAAAAAA	-3.58811
Seq2	48	55	1	AAAAAAAA	-3.58811
Seq2	49	56	1	AAAAAAAA	-3.58811
Seq2	50	57	1	AAAAAAAA	-3.58811
Seq3	3	10	0	TTTTTTTT	-3.58811
Seq3	11	18	0	TTTTTTTT	-3.58811
Seq3	10	17	0	TTTTTTTT	-3.58811
Seq3	9	16	0	TTTTTTTT	-3.58811
Seq3	8	15	0	TTTTTTTT	-3.58811
Seq3	7	14	0	TTTTTTTT	-3.58811
Seq3	6	13	0	TTTTTTTT	-3.58811
Seq3	5	12	0	TTTTTTTT	-3.58811
Seq3	4	11	0	TTTTTTTT	-3.58811
Seq3	52	59	0	TTTTTTTT	-3.58811
Seq3	51	58	0	TTTTTTTT	-3.58811
Seq3	50	57	0	TTTTTTTT	-3.58811
Seq3	49	56	0	TTTTTTTT	-3.58811
Seq3	48	55	0	TTTTTTTT	-3.58811
Seq3	47	54	0	TTTTTTTT	-3.58811
Seq3	46	53	0	TTTTTTTT	-3.58811
Seq3	45	52	0	TTTTTTTT	-3.58811
Seq3	44	51	0	TTTTTTTT	-3.58811
Seq3	43	50	0	TTTTTTTT	-3.58811
Seq3	42	49	0	TTTTTTTT	-3.58811
Seq3	41	48	0	TTTTTTTT	-3.58811
Seq3	40	47	0	TTTTTTTT	-3.58811
Seq3	39	46	0	TTTTTTTT	-3.58811
Seq3	38	45	0	TTTTTTTT	-3.58811
Seq3	37	44	0	TTTTTTTT	-3.58811
Seq3	36	43	0	TTTTTTTT	-3.58811
Seq3	35	42	0	TTTTTTTT	-3.58811
Seq3	34	41	0	TTTTTTTT	-3.58811
Seq3	33	40	0	TTTTTTTT	-3.58811
Seq3	32	39	0	TTTTTTTT	-3.58811
Seq3	31	38	0	TTTTTTTT	-3.58811
Seq3	22	29	0	TTTTTACT	-3.58811
Seq3	19	26	0	TTTTTTTT	-3.58811
Seq3	18	25	0	TTTTTTTT	-3.58811
Seq3	17	24	0	TTTTTTTT	-3.58811
Seq3	16	23	0	TTTTTTTT	-3.58811
Seq3	15	22	0	TTTTTTTT	-3.58811
Seq3	14	21	0	TTTTTTTT	-3.58811
Seq2	35	42	1	AAAAAAAA	-3.58811
Seq2	34	41	1	AAAAAAAA	-3.58811
Seq2	33	40	1	AAAAAAAA	-3.58811
Seq2	32	39	1	AAAAAAAA	-3.58811
Seq2	31	38	1	AAAAAAAA	-3.58811
Seq2	27	34	1	ACTGAAAA	-3.58811
Seq2	20	27	1	AAAAAAAA	-3.58811
Seq2	19	26	1	AAAAAAAA	-3.58811
Seq2	18	25	1	AAAAAAAA	-3.58811
Seq2	17	24	1	AAAAAAAA	-3.58811
Seq2	16	23	1	AAAAAAAA	-3.58811
Seq2	15	22	1	AAAAAAAA	-3.58811
Seq2	14	21	1	AAAAAAAA	-3.58811
Seq2	13	20	1	AAAAAAAA	-3.58811
Seq2	12	19	1	AAAAAAAA	-3.58811
Seq2	11	18	1	AAAAAAAA	-3.58811
Seq2	10	17	1	AAAAAAAA	-3.58811
Seq2	9	16	1	AAAAAAAA	-3.58811
Seq2	8	15	1	AAAAAAAA	-3.58811
Seq2	7	14	1	AAAAAAAA	-3.58811
Seq2	6	13	1	AAAAAAAA	-3.58811
Seq2	5	12	1	AAAAAAAA	-3.58811
Seq2	4	11	1	AAAAAAAA	-3.58811
Seq2	3	10	1	AAAAAAAA	-3.58811
Seq2	2	9	1	AAAAAAAA	-3.58811
Seq2	1	8	1	AAAAAAAA	-3.58811
Seq2	0	7	1	AAAAAAAA	-3.58811
Seq2	39	46	1	AAAAAAAA	-3.58811
Seq2	36	43	1	AAAAAAAA	-3.58811
Seq2	40	47	1	AAAAAAAA	-3.58811
Seq2	42	49	1	AAAAAAAA	-3.58811
Seq2	38	45	1	AAAAAAAA	-3.58811
Seq2	37	44	1	AAAAAAAA	-3.58811
Seq2	41	48	1	AAAAAAAA	-3.58811
Seq2	22	29	0	AAAAAACT	-3.58811
Seq3	27	34	1	ACTGTTTT	-3.58811
Seq3	36	43	1	TTTTTTTT	-5.19755
Seq2	46	53	0	AAAAAAAA	-5.19755
Seq3	37	44	1	TTTTTTTT	-5.19755
Seq2	7	14	0	AAAAAAAA	-5.19755
Seq3	38	45	1	TTTTTTTT	-5.19755
Seq2	40	47	0	AAAAAAAA	-5.19755
Seq3	39	46	1	TTTTTTTT	-5.19755
Seq2	6	13	0	AAAAAAAA	-5.19755
Seq2	8	15	0	AAAAAAAA	-5.19755
Seq3	35	42	1	TTTTTTTT	-5.19755
Seq3	34	41	1	TTTTTTTT	-5.19755
Seq2	9	16	0	AAAAAAAA	-5.19755
Seq3	33	40	1	TTTTTTTT	-5.19755
Seq2	47	54	0	AAAAAAAA	-5.19755
Seq3	32	39	1	TTTTTTTT	-5.19755
Seq2	10	17	0	AAAAAAAA	-5.19755
Seq3	31	38	1	TTTTTTTT	-5.19755
Seq2	39	46	0	AAAAAAAA	-5.19755
Seq2	11	18	0	AAAAAAAA	-5.19755
Seq2	48	55	0	AAAAAAAA	-5.19755
Seq2	42	49	0	AAAAAAAA	-5.19755
Seq3	52	59	1	TTTTTTTT	-5.19755
Seq2	0	7	0	AAAAAAAA	-5.19755
Seq3	51	58	1	TTTTTTTT	-5.19755
Seq3	50	57	1	TTTTTTTT	-5.19755
Seq2	1	8	0	AAAAAAAA	-5.19755
Seq3	49	56	1	TTTTTTTT	-5.19755
Seq2	43	50	0	AAAAAAAA	-5.19755
Seq3	48	55	1	TTTTTTTT	-5.19755
Seq2	2	9	0	AAAAAAAA	-5.19755
Seq3	47	54	1	TTTTTTTT	-5.19755
Seq2	41	48	0	AAAAAAAA	-5.19755
Seq3	46	53	1	TTTTTTTT	-5.19755
Seq2	3	10	0	AAAAAAAA	-5.19755
Seq3	45	52	1	TTTTTTTT	-5.19755
Seq2	44	51	0	AAAAAAAA	-5.19755
Seq3	44	51	1	TTTTTTTT	-5.19755
Seq2	4	11	0	AAAAAAAA	-5.19755
Seq3	43	50	1	TTTTTTTT	-5.19755
Seq3	42	49	1	TTTTTTTT	-5.19755
Seq2	5	12	0	AAAAAAAA	-5.19755
Seq3	41	48	1	TTTTTTTT	-5.19755
Seq2	45	52	0	AAAAAAAA	-5.19755
Seq3	40	47	1	TTTTTTTT	-5.19755
Seq2	24	31	0	AAAACTGA	-5.19755
Seq3	15	22	1	TTTTTTTT	-5.19755
Seq3	0	7	1	TTTTTTTT	-5.19755
Seq3	14	21	1	TTTTTTTT	-5.19755
Seq3	13	20	1	TTTTTTTT	-5.19755
Seq3	12	19	1	TTTTTTTT	-5.19755
Seq2	31	38	0	AAAAAAAA	-5.19755
Seq3	11	18	1	TTTTTTTT	-5.19755
Seq3	1	8	1	TTTTTTTT	-5.19755
Seq3	10	17	1	TTTTTTTT	-5.19755
Seq2	32	39	0	AAAAAAAA	-5.19755
Seq3	9	16	1	TTTTTTTT	-5.19755
Seq2	36	43	0	AAAAAAAA	-5.19755
Seq3	8	15	1	TTTTTTTT	-5.19755
Seq2	33	40	0	AAAAAAAA	-5.19755
Seq3	7	14	1	TTTTTTTT	-5.19755
Seq3	2	9	1	TTTTTTTT	-5.19755
Seq3	6	13	1	TTTTTTTT	-5.19755
Seq2	34	41	0	AAAAAAAA	-5.19755
Seq3	5	12	1	TTTTTTTT	-5.19755
Seq3	4	11	1	TTTTTTTT	-5.19755
Seq2	35	42	0	AAAAAAAA	-5.19755
Seq3	3	10	1	TTTTTTTT	-5.19755
Seq3	29	36	1	TGTTTTTT	-5.19755
Seq2	12	19	0	AAAAAAAA	-5.19755
Seq2	13	20	0	AAAAAAAA	-5.19755
Seq2	49	56	0	AAAAAAAA	-5.19755
Seq2	14	21	0	AAAAAAAA	-5.19755
Seq2	38	45	0	AAAAAAAA	-5.19755
Seq2	15	22	0	AAAAAAAA	-5.19755
Seq2	50	57	0	AAAAAAAA	-5.19755
Seq2	16	23	0	AAAAAAAA	-5.19755
Seq2	17	24	0	AAAAAAAA	-5.19755
Seq2	51	58	0	AAAAAAAA	-5.19755
Seq2	18	25	0	AAAAAAAA	-5.19755
Seq2	37	44	0	AAAAAAAA	-5.19755
Seq2	19	26	0	AAAAAAAA	-5.19755
Seq3	19	26	1	TTTTTTTT	-5.19755
Seq2	52	59	0	AAAAAAAA	-5.19755
Seq3	18	25	1	TTTTTTTT	-5.19755
Seq2	20	27	0	AAAAAAAA	-5.19755
Seq3	17	24	1	TTTTTTTT	-5.19755
Seq3	16	23	1	TTTTTTTT	-5.19755
Seq3	20	27	0	TTTTTTTA	-5.19755
Seq3	24	31	0	TTTACTGT	-5.19755
Seq2	29	36	1	TGAAAAAA	-5.19755
Seq3	20	27	1	TTTTTTTA	-6.80698
